src/shelly_hap_light_bulb.hpp

```diff
@@ -43,6 +43,13 @@ class RGBWLight : public Component, public mgos::hap::Service {
     float w;
   };
 
+  struct NotificationCharacteristics {
```

Comment:
why do we need a struct? are they ever used as a group? i think they are more or less independent.

Reply:
No explicit need, just use them to group, I prefer this over having 5 (1 will added later for color temp) variables

Reply:
removed